requirements.txt

```diff
@@ -0,0 +1 @@
+langchain
```

Comment:
Cool! First time to meet 🤯

Reply:
Langchain is convenient and good!
But there's still many bugs and poor documentation issues. : (